config/local_restrictions.yml

```diff
@@ -2033,3 +2033,21 @@ E07000202:
     - alert_level: 2
       start_date: 2020-12-02
       start_time: "00:01"
+E06000052:
+  name: Cornwall Council
+  restrictions:
+    - alert_level: 1
+      start_date: 2020-12-02
+      start_time: "00:01"
+E06000046:
+  name: Isle of Wight Council
+  restrictions:
+    - alert_level: 1
+      start_date: 2020-12-02
+      start_time: "00:01"
+E06000053:
+  name: Council of the Isles of Scilly
+  restrictions:
+    - alert_level: 1
+      start_date: 2020-12-02
+      start_time: "00:01"
```

Comment:
As a future step should we consider having LocalRestriction raise an exception if there is an area here which doesn't have a restriction? It feels like there's places where the current_restiction method could return nil based on the data in this YAML file but is actually something that would break the application logic.

Reply:
I will add a card to the backlog!